resources/sass/pages/_landing.scss

```diff
@@ -0,0 +1,322 @@
+$text-shadow: 0 0 100px $color-black-true;
+$box-shadow: 0 5px 150px $color-black-true;
+
+.back-to-top {
+    position: fixed;
+    bottom: 1rem;
+    right: 1rem;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+
+    z-index: 10;
+
+    svg {
+        transform: rotate(180deg);
+        stroke: $color-white;
+        fill: $color-raspberry-red;
+    }
+}
+
+.landing {
+    height: calc(100vh - 60px); // 60px - nav height
+    overflow-y: scroll;
+    scroll-snap-type: y mandatory;
+}
+
+.content {
+    height: 100%;
+    max-width: $page-margins;
+    padding: 0 1rem;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    color: $color-white;
+    text-transform: uppercase;
+    .hero-title {
+        margin-bottom: 2.5rem;
+    }
+
+    p {
+        font-size: $font-size-md;
+        font-weight: bold;
+        text-shadow: $text-shadow;
+    }
+    h1 {
+        line-height: 100%;
+        font-size: $font-size-xxl;
+        font-family: $font-display;
+        text-shadow: $text-shadow;
+        text-align: center;
+    }
+}
+.landing-section-wrapper {
+    height: calc(100vh - 60px); // 60px - nav height
+    position: relative;
+    scroll-snap-align: start none;
+    background: linear-gradient(
+        to bottom,
+        rgba(0, 0, 0, 0.7) 0%,
+        rgba(0, 0, 0, 0.5) 50%,
+        transparent 100%
+    );
+
+    .screen-image {
+        z-index: -1;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        object-position: center;
+        position: absolute;
+
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
+    }
+}
+
+.no-fade {
+    background: none;
+}
+
+.user-guidance {
```

Comment:
Mazāku prasītos

Reply:
Tā kā font sizes ir mazāki un arī icons, šis automātiski kļuvis mazāks.